Automattic/wp-calypso
# Add Hotjar to Signup flow

#### Proposed Changes

* Add Hotjar tracking to the signup flow. 

#### Testing Instructions



* Go to `/start` and check browser network requests, verify that you see a request to hotjar.
* The request should only be fired in the `onboarding` signup flow, and not any other flow.
